web-admin/src/routes/+layout.ts

```diff
@@ -103,6 +108,7 @@ export const load = async ({ params, url, route }) => {
 
     return {
       organizationPermissions,
+      organizationQuotas,
```

Comment:
It looks like `organizationQuotas` is only used in the `PlanQuotas.svelte` component. Rather than prop drill `organizationQuotas` through the app, I think it'd be cleaner to just pull the info from the TanStack Query cache in `PlanQuotas.svelte`.